src/applications/facility-locator/containers/VAMap.jsx

```diff
@@ -29,11 +29,14 @@ import {
 } from '../constants';
 import { areGeocodeEqual, setFocus } from '../utils/helpers';
 import { facilityLocatorShowCommunityCares } from '../utils/selectors';
-import { isProduction } from 'platform/site-wide/feature-toggles/selectors';
+import {
+  isProduction,
```

Comment:
@department-of-veterans-affairs/facility-locator-frontend is this `isProduction` flag still necessary? Looks like there's a `facilityLocatorShowCommunityCares` that can probably be removed as well because I'm assuming we always want to show Community Cares now.

Reply:
Good point. I'm, uh, gonna leave that out of this PR, though. :wink: